nodejs-diff/nodejs-diff.spec

```diff
@@ -0,0 +1,62 @@
+%global npm_name diff
+%global enable_tests 1
+
+%{?nodejs_find_provides_and_requires}
```

Comment:
This one can be removed, there's an identical macro in line 22. Other than that :+1: